ci/SetupDevEnv/README.md

```diff
@@ -0,0 +1,11 @@
+# Usage
+
+The application `SetupDevEnv` takes a `Project.toml` and add all dependencies which match a filter rule as develop version to the current julia environment. The path to the `Project.toml` is set via first application parameter.
```

Comment:
The application `SetupDevEnv.jl` takes a `Project.toml` and adds all dependencies which match a filter rule as the development version to the current Julia environment. The first application parameter sets the `Project.toml` path.